lessons/1-variables.md

```diff
@@ -1 +1,144 @@
-## Lesson 1 - Variables
+## Lesson 1 - Variables - DataTypes
+
+- [Variables](#variables)
+  - [Number](#number)
+  - [Text(String)](#textstring)
+  - [List(Array)](#listarray)
+  - [Boolean](#boolean)
+- [DataTypes](#datatypes)
+- [Examples](#examples)
+
+## Variables
+
+Every practical programming language has a feature to do this, called variables.
```

Comment:
What do you think about either omitting this sentence or rephrasing it to something like `Every practical programming language has a feature called variables.`